### ISSUE TITLE
Question about test score

### ISSUE THREAD

**Issue by wenhuchen** [eriche2016/image_caption_with_semantic_attenion#2]
Is this score from official test server or your own split from val2014 data. Even the best team from CodaLab does not achieve such a high CIDEr score.

**Comment by eriche2016** [comment]
No, this model is trained on COCO training dataset,  and i sampled 5000 images from validation set as testing and validation purpose respectively.  You can see Quanzeng's paper for more detail.  By the way, the reason why it is so high is that i use the groundtruth attributes as guidance.

**Comment by martini9393** [comment]
What do you mean by "using the groundtruth attributes as guidance" ?
